WPFHospitalEditor/AdditionalInformation.xaml.cs

```diff
@@ -102,8 +107,12 @@ private void createRowContent(string[] contentRows)
                 }
             }
             insertData();
+            if (this.allEquipment != null)
+            {
+                addEquipmentButton(contentRows);
+            }
```

Comment:
I guess you need to check if dict is empty.

Reply:
If a room has no equipment, response from the server is null, that's why i checked that.

Reply:
fixed